docs/usage.md

```diff
@@ -99,9 +100,11 @@ export default {
 };
 ```
 
-## With Svelte VS Code
+## With Svelte Config
 
-[svelte-vscode](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode) needs to know how its (svelte) language server should preprocess your files. This can be achieved by creating a `svelte.config.js` file at the root of your project which exports a svelte options object (similar to `svelte-loader` and `rollup-plugin-svelte`).
+Useful for [svelte-vscode](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode) which needs to know how its (svelte) language server should preprocess your files and [svelte-kit](https://github.com/sveltejs/kit) which can preprocess your svelte files prior to building.
+
+This can be achieved by creating a `svelte.config.js` file at the root of your project which exports a svelte options object (similar to `svelte-loader` and `rollup-plugin-svelte`).
```

Comment:
What do you think of the following? cc @dummdidumm 

```suggestion
Some tools of the Svelte ecosystem, such as [svelte-vscode](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode) and [svelte-kit](https://github.com/sveltejs/kit), need access to your svelte configuration so they know how to properly handle your Svelte files. This can be achieved by creating a `svelte.config.js` file at the root of your project which exports a svelte options object (similar to `svelte-loader` and `rollup-plugin-svelte`).
```

Also, do we have official documentation for the config file? I'm only finding references to the kit part of it.

Reply:
Yeah that text sounds good. This is what I meant by "introductionary", sorry if I didn't express that clearly enough.

Reply:
> we don't, but it's a bit out of scope for this specific PR.

Yeah, just wanted to know to link to it if it existed 👍

Reply:
So I guess is the formal definition:

https://github.com/sveltejs/kit/blob/master/packages/kit/src/core/load_config/options.js

It would be good if we could generate documentation based on this file, as an official "source of truth"